src/main/scala/jp/co/septeni_original/sbt/dao/generator/SbtDaoGenerator.scala

```diff
@@ -293,6 +293,7 @@ trait SbtDaoGenerator {
             rs.getString("COLUMN_NAME"),
             rs.getString("TYPE_NAME"),
             rs.getString("IS_NULLABLE") == "YES",
+            rs.getString("IS_AUTOINCREMENT") == "YES",
```

Comment:
[nits] [Oracle9iでは例外が投げられる](https://www.ne.jp/asahi/hishidama/home/tech/java/jdbc_meta.html#h_ResultSetMetaData) という話もあり、元々ここだけ例外捕まえてたこともあるので、念のために Try でラップするのは如何でしょうか？
```suggestion
            Try(rs.getString("IS_AUTOINCREMENT") == "YES").getOrElse(false),
```